src/test/java/com/jcabi/immutable/ArrayTest.java

```diff
@@ -197,4 +199,19 @@ public void makesSubList() throws Exception {
         );
     }
 
+    /**
+     * Array must be independent from data we passed to its ctor.
+     * @throws Exception If some problem inside
+     * @checkstyle MagicNumberCheck (10 lines)
+     */
+    @Test
+    public void isIndependentFromCtorParam() throws Exception {
+        final Integer[] ints = new Integer[] {1, 2, 3};
+        final Array<Integer> array = new Array<Integer>(ints);
+        ints[1] = 0;
+        Assert.assertTrue(
+                Arrays.equals(array.toArray(), new Integer[]{1, 2, 3})
```

Comment:
@nesterov-n I might be wrong, but the indentation is wrong here. Please, run build with `-Pqulice` to check